render-test/android/app/build.gradle

```diff
@@ -33,6 +33,19 @@ android {
             path '../../../next/CMakeLists.txt'
         }
     }
+    signingConfigs {
```

Comment:
there is no need to actually provide a signing configuration, we can get away with signing with a debug certificate.
See https://github.com/mapbox/mapbox-gl-native/blob/master/platform/android/MapboxGLAndroidSDKTestApp/build.gradle#L48 for an example.